Troubleshoot/TroubleshootError.ps1

```diff
@@ -80,7 +93,6 @@ if (($null -eq $azureRmProfileModule) -or ($null -eq $azureRmResourcesModule) -o
                 Write-Host("Close other powershell logins and try installing the latest modules for AzureRM.Resoureces in a new powershell window: eg. 'Install-Module AzureRM.Resoureces -Repository PSGallery -Force'") -ForegroundColor Red 
                 exit
             }
-	
             try {
                 Write-Host("Installing AzureRM.OperationalInsights...")
```

Comment:
I think, we should install modules ONLY the ones are missing. This is same thing we discussed last week in my PR.

Reply:
updated in PR into ci_feature : abandoning this PR:
https://github.com/Microsoft/OMS-docker/pull/243